### PR TITLE
Add night mode preview for FeedScreen

### LINKED ISSUE #249: Add night mode preview for FeedScreen
## Kind (Required)

- Improvement

 ## Overview (Required)

-  The title says everything
- Probably You can do it with pass true to the theme. https://github.com/DroidKaigi/conference-app-2021/blob/main/uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt#L272

 ## Links	

-

### PR DESCRIPTION
**Pull request by BokaDaiki**
## Issue
- close #249 

## Overview (Required)
- Add ConferenceAppFeederTheme for FeedItemRow

## Links
-

## Screenshot
I'm sorry. The preview did not work well due to a machine spec issue.

### CODE REVIEW

**Comment by tomoya0x00** [comment]
👀

**Comment by tomoya0x00** [comment]
@BokaDaiki 
Thank you for your tackling this issue 🙇 
But could we have a night mode composable preview for FeedScreen?

In this case, please refer to `PreviewFeedScreen` function.

```
@Preview(showBackground = true)
@Composable
fun PreviewFeedScreen() {
    AppThemeWithBackground {
        ProvideFeedViewModel(viewModel = fakeFeedViewModel()) {
            FeedScreen(
                selectedTab = FeedTab.Home,
                onSelectedTab = {},
                onNavigationIconClick = {
                }
            ) { feedItem: FeedItem ->
            }
        }
    }
}
```

ATTENTION:
We need NIGHT mode preview.
I think if passed `Theme.DARK` param to `AppThemeWithBackground`, it can.

Thanks!

**Comment by BokaDaiki** [comment]
I'll try

**Comment by BokaDaiki** [comment]
Fixed. Please check. Thank you.

**Comment by tomoya0x00** [comment]
@BokaDaiki 
Sorry for our late reply. 
But could you `ADD` night mode preview function? Such as `PreviewDarkFeedScreen`.

Sorry for our misleading explanation.

**Comment by BokaDaiki** [comment]
@tomoya0x00 
Thank you. I'll try.

**Comment by BokaDaiki** [comment]
Add DarkMode Preview. Please check. Thank you.

File: uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt [hunk 1]
```diff
@@ -423,6 +425,22 @@ fun PreviewFeedScreen() {
     }
 }
 
+@Preview(showBackground = true)
+@Composable
+fun PreviewDarkFeedScreen() {
+    AppThemeWithBackground {
```
Comment (tomoya0x00) [inline uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt:431]: Could you pass `Theme.DARK` param to `AppThemeWithBackground`?

File: uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt [hunk 2]
```diff
@@ -45,6 +45,8 @@ import dev.chrisbanes.accompanist.insets.toPaddingValues
 import io.github.droidkaigi.feeder.FeedContents
 import io.github.droidkaigi.feeder.FeedItem
 import io.github.droidkaigi.feeder.Filters
+import io.github.droidkaigi.feeder.PlayingPodcastState
```
Comment (tomoya0x00) [inline uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt:48]: I think this import statement is not need 👀

**Review by tomoya0x00 — CHANGES_REQUESTED** [review]
@BokaDaiki Could you check my comments? Thanks!

File: uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt (hunk 2 above)
Comment (BokaDaiki) [reply uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt:48]: that's right. I'll fix it.

**Comment by BokaDaiki** [comment]
Thank you for your comment. Fixed.

**Comment by tomoya0x00** [comment]
@BokaDaiki Thanks for your work 🙇  Could you solve https://github.com/DroidKaigi/conference-app-2021/pull/371#discussion_r598205567 too? 🙏

**Comment by BokaDaiki** [comment]
I missed the comment. Is possible.

**Comment by BokaDaiki** [comment]
Fixed. Many times Thank you.

**Review by tomoya0x00 — APPROVED** [review]
LGTM! Thank you for your contribution 🥇

**Pull request merged**